can_tools/scrapers/nytimes/nyt_cases_deaths.py

```diff
@@ -0,0 +1,274 @@
+    # https://directorsblog.health.azdhs.gov/covid-19-antigen-tests/
+    ("04", "2020-09-17", 577),
+    # https://twitter.com/AZDHS/status/1306974258124468226
+    ("04", "2020-09-18", 764),
+    # https://trello.com/c/9EbBRV1Z/472-2020-09-26-north-carolina-change-in-methodology-to-count-cases
+    ("37", "2020-09-25", 4544),
+    # https://www.al.com/news/2020/10/alabama-adds-3852-covid-cases-after-influx-of-backlogged-data-dating-back-to-june.html
+    ("01", "2020-10-23", 2565),
+]
+
+
+class NYTimesCasesDeaths(FederalDashboard, ETagCacheMixin):
+    source_name = "The New York Times"
+    source = "https://github.com/nytimes/covid-19-data"
+
+    has_location = True
+    provider: str = "nyt"
+    location_type = ""  # multiple location types, so we identify the type with a column
+    cache_file = "nyt_cases_deaths.txt"
+    file_slugs = {
+        "county": "us-counties.csv",
+        "state": "us-states.csv",
+        "nation": "us.csv",
+    }
+
+    variables = {
+        "cases": CUMULATIVE_CASES_PEOPLE,
+        "deaths": CUMULATIVE_DEATHS_PEOPLE,
+    }
+
+    def __init__(self, execution_dt: pd.Timestamp = pd.Timestamp.utcnow()):
+        # All the files appear to be updated at the same time.
+        # Use the national file for cache checking since it's the smallest
+        ETagCacheMixin.initialize_cache(
+            self,
+            cache_url=NYTIMES_RAW_BASE_URL + "us.csv",
+            cache_file=self.cache_file,
+        )
+        super().__init__(execution_dt=execution_dt)
+
+    def fetch(self) -> pd.DataFrame:
+        data = []
+        for location_type, file in self.file_slugs.items():
+            location = pd.read_csv(
+                NYTIMES_RAW_BASE_URL + file, dtype={"fips": str}
+            ).assign(location_type=location_type)
+            # Nation-level file does not have FIPS column, so create one
+            if file == "us.csv":
+                location = location.assign(fips=0)
+            data.append(location)
+        return pd.concat(data)
+
+    def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
+        data = remove_county_backfilled_cases(data, COUNTY_BACKFILLED_CASES)
+        data = remove_state_backfilled_cases(data, STATE_BACKFILLED_CASES)
+        data = remove_ma_county_zeroes_data(data)
+        return (
+            # Remove records with no FIPS codes
+            data.replace("nan", nan)
+            .loc[~data["fips"].isna(), :]
+            .pipe(
+                self._rename_or_add_date_and_location,
+                location_column="fips",
+                date_column="date",
+            )
+            .assign(location=lambda row: pd.to_numeric(row["location"]))
+            .pipe(
+                self._reshape_variables,
+                variable_map=self.variables,
+                id_vars=["location_type"],
+            )
+            # two Alaska FIPS not in our location csv
+            .query("location not in [2997, 2998]")
+        )
+
+
```

Comment:
Everything from here and below is effectively copied from [covid-data-public](https://github.com/covid-projections/covid-data-public/blob/main/scripts/update_nytimes_data.py)